tests/test.py

```diff
@@ -608,10 +608,10 @@ def test_delete(self):
         with LuxonisDataset(self.team_id, self.dataset_id) as dataset:
             for sample in dataset.fo_dataset:
                 break
-            dataset.delete([sample.id])
+            dataset.delete(["fe163bd9-6381-5533-9d41-c1735edf96d5"])
```

Comment:
Is this supposed to be hardcoded in?

Reply:
It was intentional since it should always be the same based on image bytes. However, I could also find this like I did with sample ID to ensure there are no issues with it being hardcoded.

Reply:
@kozlov721 latest commit removes this hardcode